packages/core/src/components/MediaPlayer/AudioPlayer/HeadlessAudioPlayer.tsx

```diff
@@ -70,9 +70,9 @@ const HeadlessAudioPlayer = React.forwardRef<
           playThroughEarpieceAndroid,
         });
       } catch (e) {
-        console.error(
-          "Failed to set audio mode. interruptionMode, playsInBackground, playsInSilentModeIOS, playThroughEarpieceAndroid might not be set. Failed with",
-          e
+        // Warn when trying to play in background during preview mode
+        console.warn(
+          "Background audio playback only works in published apps, not in preview mode. For iOS apps, add 'audio' under UI Background Modes in Project Settings > Apple App Store."
```

Comment:
Can we figure out which specific error is thrown that requires this message? There might be some other error in which this is not the correct message.